src/pages/version/VersionTabs.tsx

```diff
@@ -35,14 +35,14 @@ const tabMap = ({
   patchId: string;
 }) => ({
   [PatchTab.Tasks]: (
-    <Tab name="Tasks" id="task-tab" data-cy="task-tab" key="tasks-tab">
+    <Tab name="Tasks" versionId="task-tab" data-cy="task-tab" key="tasks-tab">
```

Comment:
the updates to `id` on the `<Tab />` component seem like a mistake (4 instances in this file)

Reply:
Good eye 💯